workflow-templates/differential-shellcheck.yml

```diff
@@ -0,0 +1,39 @@
+---
+
+name: Differential ShellCheck
+on:
+  push:
+  pull_request:
+    branches: [ $default-branch ]
```

Comment:
$default-branch is GitHub variable?

Reply:
Yes, but you can use it only in starter workflows (templates). From [GitHub docs](https://docs.github.com/en/actions/using-workflows/creating-starter-workflows-for-your-organization#creating-a-starter-workflow):

_If you need to refer to a repository's default branch, you can use the `$default-branch` placeholder. When a workflow is created the placeholder will be automatically replaced with the name of the repository's default branch._